src/java/org/apache/cassandra/index/sai/disk/format/IndexComponent.java

```diff
@@ -89,6 +89,19 @@
      * V2
      */
     PRIMARY_KEY_BLOCK_OFFSETS("PrimaryKeyBlockOffsets"),
+    /**
+     * V3 Block Terms Index Format Files and Meta options
+     */
+    BLOCK_BITPACKED("BlockBitPacked"), // file with many things
```

Comment:
I'm inclined to agree with this. While `// file with many things` and `// file` are descriptive to an extent they don't tell us anything about the file. At a minimum it would be nice to know what the file holds and what format it is held in.

Reply:
Updated